### ISSUE TITLE
Long text doesn't wrap

### ISSUE THREAD

**Issue by evax** [wangp/bower#13]
Long text (either in the headers or in the message view) doesn't wrap and gets truncated when it goes off screen

**Comment by wangp** [comment]
The words in the body are wrapped. It won't reflow automatically upon terminal resize; you have to refresh with '='. Maybe in the future.

Words longer than the terminal width are simply cut off. Perhaps that is what you meant?

I've implemented header wrapping as most of the code was already there. I'll push it next week.

**Comment by evax** [comment]
Yes indeed, you're right, the problem is with words longer than the terminal width (long URLs getting truncated) as well as the headers. Thanks for looking into this!

**Comment by wangp** [comment]
Long header lines will now wrap.

Breaking long words over terminal lines raises issues with searching (for substrings that across the break) and highlighting.
